src/views/community/manager/views/AddedBeneficiaryScreen.tsx

```diff
@@ -257,10 +257,14 @@ function AddedBeneficiaryScreen() {
                     {i18n.t('remove')}
                 </Button>
             )}
-            left={() => item.suspect && <WarningRedTriangle />}
+            left={() =>
+                item.suspect && <WarningRedTriangle style={{ marginTop: 14 }} />
```

Comment:
can we use vertical alignment rather than fixed margins?

Reply:
In this case, I can use at most the same approach button does within the  List.Item.

Reply:
Thing is, every time I've used fixed margin, never matched the design and it looked different on different devices.
I guess it's easier. What's yours opinion?